### DISCUSSION TITLE
Option allowing access to the back office without transmission of accesses.

### DISCUSSION THREAD

**Discussion by ARV-PS** [PrestaShop/PrestaShop#40208] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

It concerns the default configuration on the Back office.
Based on the customers' insights and security risk they run while sharing their Admin access with tech-editors.

Back-office accesses are transmitted in clear text via the support mailbox or by email.
These accesses are stored in clear text in the history of exchanges with the merchant.

### Describe the solution you'd like

The idea is to minimize the risks of a security breach in case of support intervention on merchant's back office.The solution would be to allow the connection to the back-office without access transmissions. This feature exists on ZenDesk for example (see attachment). We have the possibility to give a certain period of time during which the ZenDesk team can take control of the agent's account, simply by clicking on an access button.

This would allow the merchant to choose who would have access to the back-office, for a limited period of time, without transmitting their accesses in cleartext via email or messaging.

![image](https://user-images.githubusercontent.com/78923443/171877326-1b7e387f-91b0-4d18-bba7-1c953273ce2f.png)

Krystian Podemski told us that a module could be developed in this sense, but that it could also be a feature implemented directly in the Core.

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [x] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by hibatallahAouadni** [comment]
Hello @ARV-PS 


Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
